relay-server/Cargo.toml

```diff
@@ -97,6 +97,8 @@ reqwest = { workspace = true, features = [
 ] }
 rmp-serde = { workspace = true }
 sentry = { workspace = true, features = ["tower-http"] }
+sentry_protos = { workspace = true }
+prost-types = { workspace = true }
```

Comment:
We should gate these behind the processing feature, no need to have them as a dependency when we don't build processing.

Can set this to optional then add `dep:sentry_protos` etc. to the feature above.